cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/servlet/UserTaskManagerTest.java

```diff
@@ -71,6 +71,22 @@ public void testCreateUserTask() {
     Assert.assertEquals(userTaskHeaderValue.getValue(), testUserTaskId.toString());
     Assert.assertEquals(future, future3);
 
+    EasyMock.reset(mockHttpServletResponse);
+    // test-case: for sync task, UserTaskManager does not create mapping between request URL and UUID.
+    HttpServletRequest mockHttpServletRequest3 = prepareRequest(null, null, "test_sync_request", Collections.emptyMap());
+    userTaskManager.getOrCreateUserTask(mockHttpServletRequest3, mockHttpServletResponse, uuid -> future, 0, false).get(0);
+
+    // mockHttpServletRequest3 does not contain User-task-id, so the only place to retrieve non-null user-task-id is _sessionKeyToUserTaskIdMap.
```

Comment:
Why do we talk about the current implementation details of a different class's variable in this test case -- i.e. `_sessionKeyToUserTaskIdMap`? When we change the implementation or even the name of `_sessionKeyToUserTaskIdMap`, are we supposed to find this comment and fix it? Or is it acceptable for this comment be stale in that case?

Reply:
`_sessionKeyToUserTaskIdMap` is a member variable of `UserTaskManager`, so I think it's fine to mention it here as the test is trying to verify a implementation detail. Perhaps I can put `{@link #_sessionKeyToUserTaskIdMap}` instead in the comment? So IntelliJ will pickup the refactor. Alternatively I can call it 'session to UUID map (currently called 
_sessionKeyToUserTaskIdMap)' in UserTaskManager so it's more stale proof.